frontend/src/components/core/Sidebar/styled-components.ts

```diff
@@ -65,43 +68,14 @@ export interface StyledSidebarNavItemsProps {
 
 export const StyledSidebarNavItems = styled.ul<StyledSidebarNavItemsProps>(
   ({ isExpanded, isOverflowing, hasSidebarElements, theme }) => {
-    const isExpandedMaxHeight = isExpanded ? "75vh" : "33vh"
-    const maxHeight = hasSidebarElements ? isExpandedMaxHeight : "100vh"
+    const isExpandedPaddingBottom = isExpanded ? theme.spacing.threeXS : 0
```

Comment:
Nit: Looks like `isOverflowing` prop is no longer used and can be removed from `StyledSidebarNavItems`

Reply:
Thanks! Updated on [6919ec3](https://github.com/streamlit/streamlit/pull/6050/commits/6919ec3e85adc6a8573561634ed007127edb35e1)